trident/ion_balance.py

```diff
@@ -245,7 +245,8 @@ def add_ion_fields(ds, ions, ftype='gas',
     # If line_database is set, then use the underlying file as the line list
     # to select ions from.
     if line_database is not None:
-        line_database = LineDatabase(line_database)
```

Comment:
I believe this is leftover in your stack from previous changes you've made on LineTools.  Is it possible to strip this out so we just focus on the Firefly functionality?